.eslintrc

```diff
@@ -52,5 +52,13 @@
     "@babel/array-bracket-newline": "off",
     "@babel/brace-style": "off",
     "@babel/semi": "warn"
-  }
+  },
+  "overrides": [
+    {
+      "files": ["*.ts", "*.tsx"],
+      "rules": {
+        "no-undef": "off"
+      }
+    }
+  ]
```

Comment:
This is a copy/paste from https://github.com/DataBiosphere/duos-ui/pull/2512/files#diff-e7e195d3d8fa5b82fa51ed952f3a179d429cbc46c0432ce1f7b357c54c162822